unitstatushandler/unitstatushandler.go

```diff
@@ -185,6 +194,31 @@ func (instance *Instance) Close() (err error) {
 	return aoserrors.Wrap(err)
 }
 
+// SendInitialUnitStaus send initial unit status.
+func (instance *Instance) SendInitialUnitStaus() error {
```

Comment:
Should be `SendUnitStratus`. This function should be called each time we connect to the cloud. And it should send current status if run status was received before. It is needed for startup case only. Thus, it should not init current status.

Actually, the problem is deeper. I created task for it to not block this MR (https://jiraeu.epam.com/browse/EPMDAEPR-2209).

Reply:
Could you keep old name of this function? `SendUnitStatus`. The name of function should represent what it is actually doing rather than how to you are using it.